### PR TITLE
Bring forward SPS marketplace adjustments

### PR DESCRIPTION
**Pull request by jpl-btlunsfo**
## Purpose

- Bring the extra bits SPS requires to safely install from the Unity Marketplace into the develop branch.

## Proposed Changes

- ADD some extra variables (unused) required by the management console's installation procedures
- CHANGE some of the module source references to work inside the management console (ssh vs https fetch issues)

## Issues

- unity-sds/unity-sps#59

## Testing

- Currently deployed in `unity-venue-dev` under `btl-df1`/`dev` (standard password)
  - http://btl-df1-dev-httpd-alb-1474979360.us-west-2.elb.amazonaws.com:8080/btl-df1/dev/sps/
  - https://www.dev.mdps.mcp.nasa.gov:4443/btl-df1/dev/sps/

### CODE REVIEW

**Comment by jpl-btlunsfo** [comment]
Just to note, this PR is actually also including the changes from #228, as they're necessary for the installation's proxies (though #228's changes arent merged yet)

File: terraform-unity/modules/terraform-unity-sps-eks/main.tf
Comment (jpl-btlunsfo) [inline terraform-unity/modules/terraform-unity-sps-eks/main.tf:12]: At present, this is pinned to a temporary tag (`59-sps-eks-marketplace-adjustments`), but I'm going to work with Galen on getting those changes into main, hopefully in the next day or two.

**Review by LucaCinquini — CHANGES_REQUESTED** [review]
Hi @jpl-btlunsfo : I was able to deploy from this branch and execute the DAGs successfully. Only question: what are the new parameters needed by the Management Console? In particular, can we set "deployment_name" to be the same as "project", so we don't have to define the same value twice for 2 different variables?

**Comment by jpl-btlunsfo** [comment]
> what are the new parameters needed by the Management Console

The management console, when installing terraform modules/applications, does a really odd force-injection of variables [here](https://github.com/unity-sds/unity-management-console/blob/main/backend/internal/terraform/management.go#L194-L200). I _think_ the terraform call includes some 'strict' setting, as the absence of any of the variables included there in the actual terraform config throws a fatal error (in the management console output at least).

> can we set "deployment_name" to be the same as "project", so we don't have to define the same value twice for 2 different variables

`deployment_name` is never actually used by the SPS terraform configuration (nor are `tags` or `install_prefix`). It _does_ get injected by the management console though?

I guess we could set its default to something simple (like a `""` empty string or something), but it definitely needs to be its own separate variable, if only to keep the management console's installer code happy.

**Comment by LucaCinquini** [comment]
@jpl-btlunsfo : ok, can you set the default value of "deployment_name" to be ""? Then we can merge these changes.

File: terraform-unity/modules/terraform-unity-sps-eks/main.tf
Comment (jpl-btlunsfo) [reply terraform-unity/modules/terraform-unity-sps-eks/main.tf:12]: right, these changes have been merged (and `59-sps-eks-marketplace-adjustments` cleaned up/deleted). We'll need to ask CS for a new tag/make a new tag for ourselves to track on before I can fix/merge this PR for good, I think.

Comment (jpl-btlunsfo) [reply terraform-unity/modules/terraform-unity-sps-eks/main.tf]: updated to new tag!

**Comment by jpl-btlunsfo** [comment]
tag updated, just need an approval to merge (will hold merging until I run another test-deploy).

**Review by LucaCinquini — APPROVED** [review]
Thanks, approved.

**Comment by jpl-btlunsfo** [comment]
No issues on personal deploy!

**Pull request merged**
